DMTR-391.tex

```diff
@@ -40,8 +45,1000 @@
 
 \maketitle
 
-% ADD CONTENT HERE
-% You can also use the \input command to include several content files.
+In this Report, we characterize the performance of the Rubin Observatory Science Pipelines Version 24.1.0. We illustrate the performance via metrics that are measured on the HSC-RC2 dataset. RC2 consists of 3 tracts of data taken from the HSC-SSP survey, and selected to provide a means of testing various ``pathological'' cases (e.g., difficult astrometric solutions, extremely good seeing that does not provide a well-sampled PSF, difficult fields for deblending, and large galaxies, among others). These three tracts each contain between 112--149 visits split between the HSC-G, HSC-R, HSC-I, HSC-Z, and HSC-Y (\emph{grizy}) filters.
+
+Following the switch to Generation 3 (``Gen 3'') Middleware that took place in Version 23.0.0 of the Science Pipelines, Version 24.1.0 contains a number of updates to the Gen 3 Middleware. Other important changes in the v24 release include the adoption of the Scarlet Lite deblender for all measurements on coadds, changing the default PSF modeling algorithm to Piff, producing a repeatable selection of stars for PSF modeling, and matching the background levels of adjacent amplifiers during ISR tasks. For more details on the major new features, see the \href{https://pipelines.lsst.io/releases/v24_0_0.html#release-latest}{release notes for v24}.
+
+All metrics reported here were calculated using the \href{https://github.com/lsst/faro}{faro} metric calculation package, which is part of the standard pipeline builds. All of the underlying algorithms to calculate metrics within \texttt{faro} are the same as they were in v23.0.0 of the Science Pipelines, so any differences between metrics from the v23 and v24 releases are due to changes in the underlying pipelines.
+
+The metric calculation pipelines from \texttt{faro} were run on the RC2 tracts to derive the photometric, astrometric, and shape metrics that are reported here. We exclude the two astrometry metrics (AM3 and AF3) that concern residuals on 200-arcminute scales, since the individual tracts of RC2 do not span large enough spatial scales to enable these measurements.
+
+For comparison, we provide the \SRD required ``design'' value of each Key Performance Metric (KPM) as defined in the Science Requirements Document \citedsp{LPM-17}. For the ellipticity correlation metrics, there are specifications only for \emph{r} and \emph{i} bands. The \emph{ugzy}-band measurements are of interest primarily for historical tracking.
+
+Some KPMs (e.g., PF1, AF1, AF2) involve thresholds that are different for ``design'', ``minimum'', and ``stretch'' specifications. Metrics in this report are all compared to the ``design'' thresholds. The assessment of these KPMs would be different if evaluated against different thresholds.
```

Comment:
It might be interesting to add in the minimum spec for cases where we do not yet meet the design spec. For many metrics we will not achieve design goals until well into the survey.